_health-care/_js/_components/name-and-general-info-section.jsx

```diff
@@ -24,6 +25,8 @@ class NameAndGeneralInfoSection extends React.Component {
                   onUserInput={(update) => {this.props.onStateChange('mothersMaidenName', update);}}/>
               <SocialSecurityNumber ssn={this.props.data.socialSecurityNumber}
                   onUserInput={(update) => {this.props.onStateChange('socialSecurityNumber', update);}}/>
+              <GenderInput name={this.props.data.gender}
```

Comment:
Also, `this.props.data.gender` is referencing the `applicationData` specified in `health-care-app.jsx`. I'm assuming if gender isn't required we don't need to set it in `applicationData`? And this initially will just be undefined? Perhaps @awong-dev can weigh in here.